### ISSUE TITLE
Frequent xruns with MOTU Ultralite AVB with jack2 - no issues with jack1

### ISSUE THREAD

**Issue by siborg** [jackaudio/jack2#337]
I am experiencing frequent xruns during both playback and recording when using MOTU Ultralite AVB interface (via USB). Light load on i7 cpu - tried both with rt and non rt kernel. 

I am experiencing no issues whatsoever when using jack1. (except that Bitwig does not support it).

Happy to help to gather intel to fix this issue. Please let me know what information/debug/logs you'd want.

**Comment by ahellquist** [comment]
I have x-runs all the time on my laptop (old i7 processor) running lightly.
Have not tried jack1 since my x-runs are not audible and I mostly use the laptop for hosting lv2 and VST instruments for live gigging.

I have pinned the CPU freq and it's prio and Im using kxstudio low latency kernel.

I find it strange that my box have so much x-runs and I did not have these issues when using the onboard Intel sound card. Now I am running one or two Ultralite AVB cards but not latest released 1.3.4 driver but one of the last betas of 1.3.4 (touch console beta)
The last released 1.3.4 caused sample rate drifting issues.

**Comment by siborg** [comment]
I am still getting xruns, sometimes audible, sometimes not. 
I am running 1.3.2+520 as 1.3.4 was even worse for me.

**Comment by ctsexton** [comment]
Yeah I found 1.3.4 unusable but 1.3.2+520 works well for me without xruns for me on Jack2.

**Comment by ahellquist** [comment]
I am running firmware 1.3.4+558 (beta) that has the new touch console and it's beeing quite stable. The release 1.3.4 was not working for me. 1.3.2+520 was great as well.

For more than 18 channels over USB, you need to go back to earlier firmware where this was configurable. Just for the record and slightly of topic.

**Comment by ctsexton** [comment]
Oh I was trying to run 24 channels on 1.3.4 (successfully doing that on 1.3.2+520). I will try it again with 18!

**Comment by siborg** [comment]
Ahellquist: Do you mind sharing your configuration? Distro, kernel, desktop environment, realtime or not, motherboard model? 

I am starting to suspect that my aging Asus board might be the problem here. 

1.3.4 (non beta) is unusable for me  - the audio crops and crackles constantly.
